test/trajectory-subsetter/utilities.py

```diff
@@ -0,0 +1,79 @@
+""" A module containing utility functionality used by the Trajectory Subsetter
+    regression tests. These functions are kept out of the Jupyter notebook to
+    increase the readability of the regression test suite.
+
+"""
+from os import listdir, remove, replace
+
+from harmony import Client, Request
+from harmony.harmony import ProcessingFailedException
+import xarray as xr
+
+
+def compare_results_to_reference_file(results_file_name: str,
```

Comment:
I think it's fine to leave it like it is for now, but I would consider how we could dry it up in the future.

Reply:
This is definitely a reasonable thought. I'm tempted to say "this PR is already a bit meaty, let's defer". But I definitely hear you.

I think there's a little bit of trickiness in making it so that the tests can access utilities when running both from the built Docker image (where we could have a single `utilities.py` that gets copied into the images) and running them from a local notebook server. I'll think about this a bit more and try to address it in the future.